### ISSUE TITLE
Media encrypted with kbpgp showing corruption

### ISSUE THREAD

**Issue by steverecio** [keybase/kbpgp#180]
I'm encrypting a number of files using kbpgp and decrypting them with GPG. While ascii text file encryption/decryption seems to work fine, I'm seeing file corruption with various media such as PDF files and JPEGs. Perhaps this could be an issue with the file compression?

For example, below I've attached a PDF copy of Cracking the Coding Interview as well as the corrupted version decrypted using GPG.

[Cracking the Coding Interview, 4 Edition - 150 Programming Interview Questions and Solutions.pdf](https://github.com/keybase/kbpgp/files/2669122/Cracking.the.Coding.Interview.4.Edition.-.150.Programming.Interview.Questions.and.Solutions.pdf)

[decrypted_cracking_the_coding_interview.pdf](https://github.com/keybase/kbpgp/files/2669129/decrypted_cracking_the_coding_interview.pdf)

**Comment by Owlincfr** [comment]
I don't think that an ArrayBuffer is really useful here and can, maybe, be the starting point of your trouble.

Try to replace `r.readAsArrayBuffer(file);` by `r.readAsBinaryString(file);` like the [documentation](https://keybase.io/kbpgp/docs/buffers) recommend it.

**Comment by jcherniak** [comment]
@steverecio, did you ever get this figured out?  I'm getting the same corruption issues.  The files decrypt with my PGP key fine, but are about 25-30% larger.

**Comment by steverecio** [comment]
@jcherniak No, I never got this library working. I had to switch to [OpenPGP](https://github.com/openpgpjs/openpgpjs).

**Comment by maxtaco** [comment]
Seems like you are getting a base64-encoded output if it's 25-30% larger on decryption
